styles.css

```diff
@@ -0,0 +1,294 @@
+/* stylelint-disable no-missing-end-of-source-newline */
+html {
```

Comment:
- [ ] Good job adding linters to your repo, :clap: it could be better to fix all the linters without disable them, Kindly fix that